docs/tutorial/getting-started.rst

```diff
@@ -57,21 +61,24 @@ Next, you need to initialise DSS on top of MicroK8s and prepare MLflow:
    dss initialize --kubeconfig="$(sudo microk8s config)"
 
 .. note::
-   The `dss initialize` command might take a few minutes to complete.
+
+   The initialisation might take a few minutes to complete.
+   While the process completes, you will see the following message:
```

Comment:
Nit, but when I quickly saw the word `completes` here my mind thought that this will be once the command completes and not while it's running.

I would slightly change this to `Whilte the process is in progress, you'll...`

Reply:
Changed!